### ISSUE TITLE
Is it possible to perform bulk operations, not only inserts to database?

### ISSUE THREAD

**Issue by diegoaguilar** [balderdashy/sails-mongo#304]
I know it's possible to perform multiple inserts in the way:

`Dogs.create({name: "Akira"}, {name: "Maddi"}, {name: "Risas"}).exec(...);`

However what when I want to do multiple update operations, maybe also deletes, etc. In particular I'd like to achieve something like what's intended in this [stackoverflow's post](http://stackoverflow.com/questions/31351572/).

**Comment by jaime-franco** [comment]
Someone find a way to do this?

**Comment by devinivy** [comment]
Your best bet is probably using `Collection.native()` to perform a native mongo request.  I believe this is documented in the waterline-docs repo and on the sails.js site for more information.

**Comment by sailsbot** [comment]
Thanks for posting, @diegoaguilar.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
